crates/pathfinder/fixtures/sequencer/0.10.1/deploy_account/deploy_account.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'd move this (and the response) under an `add_transaction` subdirectory, something like:
`crates/pathfinder/fixtures/sequencer/0.10.1/add_transaction/deploy_account_request.json` and `deploy_account_response.json`.

I'm suggesting this because this JSON is the representation for the `add_transaction` endpoint, and not the representation of a deploy account transaction in a block.

Reply:
Done